db.go

```diff
@@ -167,6 +167,7 @@ func (db *DB) String() string {
 
 // Open creates and opens a database at the given path.
 // If the file does not exist then it will be created automatically.
+// A mode of 0o600 "user read/write" is conventional.
```

Comment:
```suggestion
// If the file does not exist then it will be created automatically with the given mod .
```

Reply:
The suggested change here is worse. It misspells 'mode' as 'mod', but more importantly doesn't give the user a clue as to _what_ mode they should use.

Reply:
Updated the description. It's up to uses which mod to set. But for read/write transactions, the owner must have write permission on the created file.